tests/e2e/vmc_create_gc.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"os"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	ghttp "github.com/onsi/gomega/ghttp"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	gcManifest = "testing-manifests/tkg/"
+)
+
+var _ = ginkgo.Describe("Create GC", func() {
+	f := framework.NewDefaultFramework("create-gc")
+	var server *ghttp.Server
+	var pvclaim *v1.PersistentVolumeClaim
+	var statusCode int
+
+	ginkgo.BeforeEach(func() {
+
+		bootstrap()
+		_ = getNamespaceToRunTests(f)
+		server = ghttp.NewServer()
+		os.Setenv("SOME_ADDR", server.Addr())
+
+		server.AppendHandlers(
```

Comment:
what are we doing here, please add comments...

Reply:
It's not required, I have removed it.